spec/01-unit/03-conf_loader_spec.lua

```diff
@@ -427,6 +462,15 @@ describe("Configuration loader", function()
   end)
 
   describe("nginx_user", function()
+    it("is 'kong kong' by default if the kong user/group exist", function()
```

Comment:
Just curious, is this even going to be triggered on CI, not sure if the CI even has the `kong` user/group.

Not a blocker though, more for my own curiosity.

Reply:
This is not going to be triggered on CI because the CI doesn't have the `kong` user available.

But when I was testing it locally the tests didn't pass because my machine _did_ have the `kong` user, so I thought I should add the verification of whether the `kong` user exist on the test suite too. Also good for readability, for your better understanding of this patch and if in the future someone sees it too.